app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -255,17 +255,19 @@ const SendAdaPage = ({
   )
 
   const displayDropdownSelectedItem = (dropdownAssetItem: DropdownAssetItem) => {
-    const {policyId, type} = dropdownAssetItem
+    const {fingerprint, type} = dropdownAssetItem
     return (
-      <div className="wrapper">
+      <div className="wrapper flex-nowrap shrinkable">
         <FormattedAssetItem {...dropdownAssetItem}>
-          {({formattedAssetIconName}) => {
+          {({formattedHumanReadableLabelType, formattedHumanReadableLabelWithIcon}) => {
+            const isFingerprint =
```

Comment:
this seems a bit confusing - so the `fingerprint` passed may not always be a fingerprint? :thinking: I'd suggest choosing some more accurate name for that property if that's the case

Reply:
The passed value is indeed a fingerprint. But normally we'd show the fingerprint in the grey text in brackets after the label. But if the label is already a fingerprint, there is no point in showing it again after it in the grey text in brackets. I'll change it to `isLabelFingerprint`, hopefully, that's less confusing.